monitoring/monitoring/streamlit_app.py

```diff
@@ -0,0 +1,81 @@
+import streamlit as st
+
+from monitoring.utils import (
+    all_timestamps_agg,
+    all_timestamps_merchant,
+    dates,
+    last_extraction_summary,
+    last_scraping_summary,
+    products_by_category,
+    products_by_label,
+    products_unknown_label,
+)
+
+st.set_page_config(page_icon="♻️", page_title="GreenDB")
+
+
+def main() -> None:
```

Comment:
For performance reasons, you should refactor this and use the connection API just once. Especially do not call the same method twice, e.g., `last_extraction_summary `

Reply:
Refactored in [9966a3c](https://github.com/calgo-lab/green-db/pull/91/commits/9966a3c572311e3a9fb91a086fa3f12240fc0202)